### PR TITLE
fix(show-completed): hard coded state filter for show-completed checkbox

### PR DESCRIPTION
**Pull request by sanbornsen**
### Mandatory
 - [ ] What does this PR do? 
In the absence of back-end support ATM to filter out Completed work items in SCRUM, adding temporary hack at UI to filter out work items with specific states (Closed, Done, Removed)
- [ ] What issue/task does this PR references?
https://github.com/openshiftio/openshift.io/issues/3486
- [ ] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @sanbornsen for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-ui/fabric8-planner/6054e2c9adefe9b483f57024e8087572345ab7b8/test-keeper.yml).

File: src/app/components_ngrx/planner-list/planner-list.component.ts [hunk 1]
```diff
@@ -183,7 +183,19 @@ export class PlannerListComponent implements OnInit, OnDestroy, AfterViewChecked
...
         if (!queryParams.hasOwnProperty('showCompleted') && !queryParams.showCompleted) {
           this.showCompleted = false;
-          exp['$AND'].push({"state":{"$NE":"closed"}});
+          // not closed state
+          // TODO remove hard coded states and
+          // use meta-states when available
+          const notClosed = this.filterService.queryBuilder(
+            'state',
+            this.filterService.not_in_notation,
+            ['closed', 'Done', 'Removed']
```
Comment (rgarg1) [inline src/app/components_ngrx/planner-list/planner-list.component.ts]: Even though _Closed_ is not a valid state for any WIT in SCRUM, having it here has no harm. LGTM

**Comment by rgarg1** [comment]
@sanbornsen The fix works! LGTM

**Comment by rgarg1** [comment]
@jarifibrahim @michaelkleinhenz Can we merge this pls.

**Comment by jarifibrahim** [comment]
@rgarg1 Merged.

**Pull request merged**